src/ProfilesList.json

```
@@ -1,4 +1,5 @@
 [
+  "sanghdeep.json"
```

```suggestion
  "sanghdeep.json",
```
Add comma after file name.